FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -2624,8 +2610,7 @@ bool FireRenderContext::isFirstIterationAndShadersNOTCached()
 
 void FireRenderContext::updateProgress()
 {
-	long numberOfClocks = clock() - m_renderStartTime;
-	double secondsSpentRendering = numberOfClocks / (double)CLOCKS_PER_SEC;
+	double secondsSpentRendering = TimeDiffChrono(GetCurrentChronoTime(), m_renderStartTime) / 1000;
```

Comment:
Is secondsSpentRendering is supposed to be double?

Reply:
yes, I did not change here anything